cssom-1/Overview.bs

```diff
@@ -2079,6 +2098,8 @@ Setting the {{CSSStyleDeclaration/cssText}} attribute must run these steps:
  <li>Empty the <a for="CSSStyleDeclaration">declarations</a>.
  <li><a lt="Parse a CSS declaration block">Parse</a> the given value and, if the return value is not the empty list, insert the items in the list
  into the <a for="CSSStyleDeclaration">declarations</a>, in <a>specified order</a>.
+ <li>If the <a for="CSSStyleDeclaration">owner node</a> is not null, <a>set an attribute value</a> for <a for="CSSStyleDeclaration">owner node</a>
```

Comment:
I really think we should prevent this attribute set from leading to a re-parse.  It's a much simpler model than having it re-parse, because in the presence of CSP parsing from the string does NOT give the same results as the thing that was serialized to the string, in general.  And since UAs are going to optimize out the reparsing for basic sanity reasons, this is likely to lead to divergence between what the spec theoretically says (but no one would ever do) and what UAs do.

There are also issues around base URIs: set vs serialize-and-reparse treat them differently.  I did explicitly mention this in https://github.com/w3c/csswg-drafts/issues/1559 as a problem with the serialize-and-reparse approach.

Not reviewing the rest of the changes for now given that, because they will need to be modified anyway.

Reply:
Do you have any suggestion for how can we avoid serialize-and-reparse? It seems to me if we want to avoid that, we would need to duplicate many definitions from the DOM spec just for getting rid of the "run attribute change steps" part.

Is there any existing algorithm in the DOM spec helps doing that? If not, I suggest we merge this change to get the spec closer to what browsers do currently, and file a new issue to address the serialize-and-reparse issue, which may need some help from the DOM spec side.

Reply:
You could set a flag while _you_ are setting the attribute, and not do any attribute change steps while that flag is set, right?

Reply:
Hmmm, okay, fair. Thanks for the suggestion.